src/log-manager/src/nginx/nginx.conf.default

```diff
@@ -70,8 +70,17 @@ server {
     limit_except GET {
       deny all;
     }
-    default_type text/plain;
+    default_type application/octet-stream;
```

Comment:
Try https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition to make browsers turn to downloads.

Reply:
Fixed

Reply:
Let's restore the type to `text/plain` for future downstream services.

Reply:
Fixed